### ISSUE TITLE
[MDCOutlinedTextField] can't setup corner radius from new Theme. Creating this from property containerRadius.

### ISSUE THREAD

**Issue by NikitosMoskalenko** [material-components/material-components-ios#10215]
iOS(11, *)
Xcode 12.4

My code: 
    static var newShapeScheme: MDCShapeScheme {
        let cornerTreatment = MDCRoundedCornerTreatment.corner(withRadius: 0.5, valueType: .percentage)
        let shapeScheme = MDCShapeScheme()
        shapeScheme.smallComponentShape.bottomLeftCorner = cornerTreatment
        shapeScheme.smallComponentShape.topLeftCorner = cornerTreatment
        shapeScheme.smallComponentShape.topRightCorner = cornerTreatment
        shapeScheme.smallComponentShape.bottomRightCorner = cornerTreatment
        
        return shapeScheme
    }

Doesn't working for MDCOutlinedTextField, but working to MDCButton.

Can you help me?

**Comment by dineybomfim** [comment]
Exactly...  same thinking here...  if they at least make MDCOutlinedTextField open class, we could subclass

**Comment by andrewoverton** [comment]
Hi, have you tried setting the `containerRadius` property on MDCOutlinedTextField's superclass MDCBaseTextFIeld?

**Comment by prateekSekhri** [comment]
Hi, I have tried setting up the corner radius with the two properties but the view is getting distorted with a extra lines in the end. It is not setting up properly in MDCOutlinedTextField as it should in the case of UITextField with layer.cornerradius and layer.borderColor. 

    _floatTextField.leadingEdgePaddingOverride = @35;
    _floatTextField.containerRadius = 30;

Please Help. Tried everything.

<img width="406" alt="Screenshot 2022-09-13 at 3 43 16 PM" src="https://user-images.githubusercontent.com/57496241/189877350-6107b6e9-b18c-4540-b3ce-f5fce4f8087f.png">
